config/routes.rb

```diff
@@ -5,5 +5,5 @@
   # events
   get '/events', to: 'events#index'
   get '/events/:urlname/:event_id', to: 'events#show'
-  get '/events/:urlname/:event_id/random', to: 'events#random_attendee'
+  get '/events/:urlname/:event_id/random', to: 'events#random'
```

Comment:
I think this name can me discussed, neither way, to rename that route is necessary rename the template too.